app/viewmodels/govuk/RadiosFluency.scala

```diff
@@ -109,6 +109,6 @@ trait RadiosFluency {
       radios copy (attributes = radios.attributes + attribute)
 
     def inline(): Radios =
-      radios.withCssClass("govuk-radios--inline")
```

Comment:
line 90 |  ).inline()
shall we just remove this from line 90 and keep the method as is incase it's ever needed?

Reply:
https://github.com/hmrc/advance-valuation-rulings-frontend/pull/13/commits/257f36812eb4432e3e97ef8ddf246dad5d59522e